src/go/rpk/pkg/system/syslog/syslog.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// +build linux
+
+package syslog
+
+import (
+	"fmt"
+	"syscall"
+)
+
+const (
+	// Taken from `man syslog`
+
+	// Read  all messages remaining in the ring buffer, placing them in the buffer pointed
+	// to by bufp.  The call reads the last len bytes from  the  log  buffer  (nondestruc‐
+	// tively),  but  will  not  read more than was written into the buffer since the last
+	// "clear ring buffer" command.  The call returns the number of bytes read.
+	syslog_action_read_all int = 3
+
+	// This  command  returns  the  total size of the kernel log buffer.
+	syslog_action_size_buffer int = 10
+)
+
+func ReadAll() ([]byte, error) {
+	bufSize, err := syscall.Klogctl(syslog_action_size_buffer, nil)
```

Comment:
This can be https://pkg.go.dev/golang.org/x/sys@v0.0.0-20210817142637-7d9622a276b7/unix#Klogctl

Reply:
Fixed.